assets/tuned/06-cm-tuned-profiles.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: tuned-profiles
+  namespace: openshift-cluster-node-tuning
+data:
+  tuned-profiles-data: |
+    openshift: |
+      [main]
+      summary=Optimize systems running OpenShift (parent profile)
+      include=${f:virt_check:virtual-guest:throughput-performance}
+
+      [selinux]
+      avc_cache_threshold=65536
```

Comment:
we are reducing this to 8192 in 3.11

Reply:
Aah, good catch.  Well, this was meant just as a PoC, will adjust.

Reply:
Addressed by https://github.com/openshift/cluster-node-tuning-operator/pull/2